<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-3</x>
      <y>60</y>
      <width>631</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Formule Finder</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>20</width>
      <height>361</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Home">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbStats">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>stats</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbHistory">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>History</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDrivers">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Drivers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLogin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>441</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>451</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Home </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Login">
     <widget class="QLabel" name="lbLogin">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>441</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="leGebruikersnaam">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>80</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbGebruikersnaam">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Gebruikersnaam:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbWachtwoord">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Wachtwoord:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leWachtwoord">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>120</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSubmit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>190</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbWelcome">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>230</y>
        <width>451</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Update">
     <widget class="QLabel" name="lbTableSelect">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Table:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbTables">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>241</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>421</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>421</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="le1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="tl1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="le4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>60</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="tl4">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>60</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="tl2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>30</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="le2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>30</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>60</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="tl3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="le5">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>90</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="tl5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="le6">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>90</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="tl6">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>90</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pbUpdate_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>320</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbDelete">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>320</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="stats">
     <widget class="QLabel" name="pbSelect">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Topic:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbTopic">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Drivers</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ranking</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cars</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Circuits</string>
       </property>
      </item>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>441</width>
        <height>271</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_4">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>100</y>
          <width>181</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Please select a topic</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageDriver">
       <widget class="QComboBox" name="cbRacers">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>181</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lbDriver">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>181</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Select Driver:</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbPhoto">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>30</y>
          <width>91</width>
          <height>111</height>
         </rect>
        </property>
        <property name="text">
         <string>Photo</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbName">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
       <widget class="QLabel" name="invName">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>201</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>t</string>
        </property>
       </widget>
       <widget class="QLabel" name="invTeam">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>150</y>
          <width>201</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>t</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbTeam">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>130</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Team:</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbCountry">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>180</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Country:</string>
        </property>
       </widget>
       <widget class="QLabel" name="invCountry">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>200</y>
          <width>201</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>t</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbRacerPoints">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>180</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Points Racer:</string>
        </property>
       </widget>
       <widget class="QLabel" name="invRpoints">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>200</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>t</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbTPoints">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>220</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Points Team:</string>
        </property>
       </widget>
       <widget class="QLabel" name="invTPoints">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>240</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>t</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="lbIngelogd">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>390</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pbUpdate">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Update DB</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
